src/metadata.rs

```diff
@@ -314,13 +335,61 @@ mod test {
         "#
         );
 
-        let actual = metadata.to_file_contents();
+        assert_metadata_from_cargo_toml(readme, cargo_toml, expected);
+    }
 
-        assert_eq!(actual.trim(), expected.trim());
+    #[test]
+    fn test_metadata_from_cargo_toml_rst() {
+        let readme = indoc!(
+            r#"
+            # Some test package
+        "#
+        );
+
+        let cargo_toml = indoc!(
+            r#"
+            [package]
+            authors = ["konstin <konstin@mailbox.org>"]
+            name = "info-project"
+            version = "0.1.0"
+            description = "A test project"
+            homepage = "https://example.org"
+            readme = "readme.md"
```

Comment:
Shouldn't this be `readme.rst`?

Reply:
It could be but it doesn't have to be, as this test just needs to prove that the metadata gets through. I'll fix it for clarity, though.

Reply:
On second look, it doesn't actually matter because the substring is replaced by the path of the temporary file created to write the readme contents into.